src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
I'm not fully in :heart: with this attribute name, but I can't think of anything else.

Maybe @cldougl could help.

with `openreverse: false` buttons drop down (or to the right)

![image](https://cloud.githubusercontent.com/assets/6675409/18791629/8030096a-8181-11e6-9e06-31854cd5b13d.png)


with `openreverse: true` the buttons drop up (or to the left)

![image](https://cloud.githubusercontent.com/assets/6675409/18791640/8a9f5d2e-8181-11e6-9e16-45b44f0a6153.png)